crates/telio-dns/tests/nameserver.rs

```diff
@@ -311,6 +407,25 @@ async fn dns_request_local() {
     .expect("Test timeout");
 }
 
+#[tokio::test]
```

Comment:
It feels like there only negative tests but no positive tests. Is there a reason for that?